controllers/provider-openstack/example/infrastructure.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: shoot--foo--bar
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This should not be needed? Why have you added it?

Reply:
Without the clusterrole my terraformer pod can't write the state to it's configmap. Not sure if I did something wrong with the local setup here.

Reply:
OK, will fix this in the Terraformer implementation itself. Thanks